src/main/java/io/kubernetes/KabaneroClient.java

```diff
@@ -49,8 +50,11 @@
 import javax.net.ssl.TrustManager;
 import javax.net.ssl.X509TrustManager;
 
+import com.google.gson.*;
```

Comment:
lets use `org.json.JSONObject;` instead of gson.  for pom.xml
```
		<dependency>
			<groupId>org.json</groupId>
			<artifactId>json</artifactId>
			<version>20190722</version>
		</dependency>
```

Reply:
It _should_ have a way to parse the json like you are doing below